Dockerfile

```diff
@@ -46,6 +51,12 @@ RUN apk update \
 COPY --from=dist /out/restic /usr/local/bin/
 COPY --from=dist /out/LICENSE /usr/share/licenses/restic/
 
+# Install rclone.
+COPY --from=dist /go/bin/rclone.v1 /usr/local/bin/
+COPY --from=dist $GOPATH/go/src/gopkg.in/ncw/rclone.v1/COPYING /usr/share/licenses/rclone/
+
+RUN mv /usr/local/bin/rclone.v1 /usr/local/bin/rclone
```

Comment:
@alnikyur this should be done on `dist` stage. Again, there is no reason to produce redudant layer in final image.